MAUI/LinearGauge/bar-pointer.md

```diff
@@ -0,0 +1,307 @@
+
+{% highlight xaml %}
+
+<gauge:SfLinearGauge>
+                <gauge:SfLinearGauge.BarPointers>
+                    <gauge:BarPointer Value="50" PointerSize="10"/>
+                </gauge:SfLinearGauge.BarPointers>
+            </gauge:SfLinearGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+		gauge.BarPointers.Add(new BarPointer()
+		{
+			Value = 50,
+			PointerSize = 10
+		});
+		this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Change the bar pointer thickness](images/bar-pointer/bar_thickness.PNG)
+
+## Customize edge style
+
+The edge style can be changed with the [`CornerStyle`]() property of the bar pointer. The edge style can be any of the `StartCurve`, `EndCurve`, `BothCurve`, and `BothFlat` options.The default value is `BothFlat`.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <gauge:SfLinearGauge>
+                <gauge:SfLinearGauge.BarPointers>
+                    <gauge:BarPointer Value="50" PointerSize="10" 
+                                      Offset="5"
+                                      Position="Outside" CornerStyle="BothCurve"/>
+                </gauge:SfLinearGauge.BarPointers>
+            </gauge:SfLinearGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+		gauge.BarPointers.Add(new BarPointer()
+		{
+			Value = 50,
+			PointerSize = 10,
+			Offset = 5,
+			Position = GaugeElementPosition.Outside,
+			CornerStyle = CornerStyle.BothCurve
+		});
+		this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Change the bar pointer edge style](images/bar-pointer/edge_style.PNG)
+
+## Customize the position
+
+By default, the bar pointer is positioned cross to the axis. This position can be changed by the [`Position`]() property of a pointer. It is possible to position the bar pointer `Inside`, `Cross`, or `Outside` the axis. The following code sample demonstrates how to change the bar pointer position to inside the axis.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <gauge:SfLinearGauge>
+                <gauge:SfLinearGauge.BarPointers>
+                    <gauge:BarPointer Value="50" Position="Inside"/>
+                </gauge:SfLinearGauge.BarPointers>
+            </gauge:SfLinearGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+		gauge.BarPointers.Add(new BarPointer()
+		{
+			Value = 50,
+			Position = GaugeElementPosition.Inside,
+		});
+		this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Customize linear gauge for bar pointer position](images/bar-pointer/bar_pointer_change_position.PNG)
+
+## Customize the offset
+
+In addition to position the bar pointer, it is also possible to change the offset of the bar pointer. The [`Offset`]() is the distance from the axis and the cross positioned elements will not get affected by the [`Offset`]() value. The following  code sample demonstrates how to change the offset value of the bar pointer. 
```

Comment:
The [`Offset`]() value is the distance from the axis and it has no effect on cross-positioned elements.